apps/rush-lib/assets/rush-init/[dot]gitignore

```diff
@@ -55,6 +55,14 @@ jspm_packages/
 # next.js build output
 .next
 
+# Common toolchain intermediate files
+lib
+temp
+
+# Rush files
+common/temp/**
+package-deps.json
```

Comment:
This should already be covered by https://github.com/microsoft/rushstack/blob/master/apps/rush-lib/assets/rush-init/%5Bdot%5Dgitignore#L63